meta/src/main/proto/gravitino_meta.proto

```diff
@@ -105,15 +105,17 @@ message Topic {
 message User {
   uint64 id = 1;
   string name = 2;
-  repeated string roles = 3;
-  AuditInfo audit_info = 4;
+  repeated string role_names = 3;
+  repeated uint64 role_ids = 4;
```

Comment:
What is the purpose to store both role names and ids?

Reply:
Role names is used for display for users. Role Ids will avoid the issues if we drop and create a role with the same name.